test/scenarios/driver/hetznercloud/molecule/default/molecule.yml

```diff
@@ -0,0 +1,37 @@
+---
+dependency:
+  name: galaxy
+
+driver:
+  name: hetznercloud
+
+lint:
+  name: yamllint
+  options:
+    config-file: ../../../resources/.yamllint
+
+platforms:
+  - name: instance
+    server_type: cx11
+    ssh_keys:
+      # FIXME(decentral1se): change when we get a CI account
```

Comment:
@LKaemmerling, molecule needs to connect to these machines but we're going to be creating them with a TTS/HCLOUD_TOKEN flow and it is not clear to me how to create SSH keys programmatically that are not first pre-authorized with the Hetzner account.

Currently, the driver implementation assumes that the SSH key referenced here is loaded with the ssh-agent. So the CI has to have control of the private part of the SSH key. That could be available from the Travis CI configuration but we'd have to agree on a key ... unsure of how to proceed ...

Reply:
You can Send me a ssh key (having a fix key) Or you could add it via the hcloud_ssh_key ansible Module

Reply:
> hcloud_ssh_key ansible Module

Right, looks like this is the solution for it! Thanks for the guidance :+1: